packages/shylu/fastilu/src/shylu_fastic.hpp

```diff
@@ -194,8 +194,8 @@ class FastICPrec
             Kokkos::parallel_for(nnzL, copyFunc3);
             double t = timer.seconds();
 
-            std::cout << "Symbolic phase complete." << std::endl;
-            std::cout << "Init time: "<< t << "s" << std::endl;
+            //std::cout << "Symbolic phase complete." << std::endl;
```

Comment:
Do you mean just
~~~~
#ifdef SHYLU_DEBUG
std::cout  << ...;
#endif
~~~~
or should I create a new switches, like maybe FASTILU_DEBUG_OUTPUT?